### DISCUSSION TITLE
How can I put a scoring system?

### DISCUSSION THREAD

**Discussion by Kharious** [dialogic-godot/dialogic#1603] [Q&A] [answered]
Hello guys, how can I put a simple scoring system through dialogic?
For example:
Got a question right = +1 point
Missed a question = -1 point

**Comment by Jowan-Spooner** [comment, marked as answer]
Hey, this sounds like a good use of dialogic variables. The implementation will vary depending on the dialogic version you use. 
Generally you will want to create a dialogic variable (e.g. called "score"). Then you can use the set variable event to add one to this variable whenever a correct answer is selected. 

You can then use this variable outside of dialogic or reference it inside of a text event, again the implementation is different for dialogic1 and dialogic2.

**Comment by Kharious** [comment]
Jowan, Thanks for the information, but I couldn't :((((((. Do you have an explanatory video?
